server/test/helpers.go

```diff
@@ -31,6 +31,7 @@ func (f DoFunc) Do(req *http.Request) (*http.Response, error) {
 
 func SetEnvironment() {
 	os.Setenv("AZURE_SUBSCRIPTION_ID", "3f7e29ba-24e0-42f6-8d9c-5149a14bda37")
+	os.Setenv("AZURE_TENANT_ID", "0b4d9aac-c9b7-4832-9013-2ab4991d5533")
```

Comment:
Is this real tenant id? If so, we should replace it with some bogus ids.

Reply:
Fixed that.